mathesar/rpc/tables.py

```diff
@@ -0,0 +1,46 @@
+from typing import Optional, TypedDict
+
+from modernrpc.core import rpc_method, REQUEST_KEY
+from modernrpc.auth.basic import http_basic_auth_login_required
+
+from db.tables.operations.select import get_table_info
+from mathesar.rpc.exceptions.handlers import handle_rpc_exceptions
+from mathesar.rpc.utils import connect
+
+
+class TableInfo(TypedDict):
+    """
+    Information about a table.
+
+    Attributes:
+        oid: The `oid` of the table in the schema.
+        name: The name of the table.
+        schema: The `oid` of the schema where the table lives.
+        description: The description of the table.
+    """
+    oid: int
```

Comment:
I'm okay with this, but we should chat at some point with @seancolsen about preferences w.r.t. specific ID names instead of just `id` in this type of case. I went with `id` rather than `attnum` for column info, for example. I don't really have a strong preference, but we should choose one way and stick with it.

Reply:
As I'm looking at this more, I think `id` is more consistent with referring to the `schema` and using its (o)id.